cranelift-codegen/meta-python/isa/x86/encodings.py

```diff
@@ -213,6 +222,9 @@ def enc_i32_i64_ld_st(inst, w_bit, recipe, *args, **kwargs):
 # Finally, the 0xb8 opcode takes an 8-byte immediate with a REX.W prefix.
 X86_64.enc(base.iconst.i64, *r.pu_iq.rex(0xb8, w=1))
 
+# bool 0 can be written as an xor rather than mov.
+enc_both(base.bconst.b1, r.u_id_bool_z, 0x30)
```

Comment:
I'm not sure if this is actually very useful, but saw the opportunity to use the same change as a shorthand for `false`.

Should this clear the entire register for a `b1`? I just realized the other encoding rule uses `b8` for full registers rather than `b0` for byte registers.

Reply:
decided to not deal with booleans here